### PR TITLE
fix(editor): Replace v-html with custom directive to sanitize html

### PR DESCRIPTION
**Pull request by cstuncsik**
## Summary

Replace `v-html` with custom directive to sanitize html 

## Review / Merge checklist

- [x] PR title and summary are descriptive.
- [x] Tests included.
- [ ] PR Labeled with `release/backport`

### CODE REVIEW

File: packages/design-system/src/components/AskAssistantChat/__tests__/__snapshots__/AskAssistantChat.spec.ts.snap [hunk 1]
```diff
@@ -129,23 +129,9 @@ exports[`AskAssistantChat > renders chat with messages correctly 1`] = `
...
             <div
               class="assistantText"
-            >
-              <p>
-                Hi Max! Here is my top solution to fix the error in your 
-                <strong>
-                  Transform data
-                </strong>
-                 node👇
-              </p>
```
Comment (tomi) [inline packages/design-system/src/components/AskAssistantChat/__tests__/__snapshots__/AskAssistantChat.spec.ts.snap:144]: These snapshot shouldn't change, right?

File: packages/design-system/src/directives/n8n-html.test.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		return {
+			unsafeHtml:
+				'<span>text</span><a href="https://malicious.com" onclick="alert(1)">malicious</a><img alt="Ok" src="./images/logo.svg" onerror="alert(2)<script>alert(3)</script>" />',
+		};
+	},
+	template: '<div v-n8n-html="unsafeHtml"></div>',
+});
+
+describe('Directive n8n-html', () => {
+	it('should sanitize html', async () => {
```
Comment (tomi) [inline packages/design-system/src/directives/n8n-html.test.ts:16]: Would be nice to have a test that it doesn't touch safe html

Comment (cstuncsik) [reply packages/design-system/src/directives/n8n-html.test.ts:15]: Done

File: packages/design-system/src/directives/n8n-html.test.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		});
+		expect(html()).toBe(
+			'<div><span>text</span><a href="https://malicious.com">malicious</a><img alt="Ok" src="./images/logo.svg"></div>',
+		);
+	});
+
+	it('should not touch safe html', async () => {
+		const { html } = render(TestComponent, {
+			props: {
+				html: '<span>text</span><a href="https://malicious.com">malicious</a><img alt="Ok" src="./images/logo.svg" />',
```
Comment (tomi) [inline packages/design-system/src/directives/n8n-html.test.ts:33]: This looks a bit odd if this is supposed to be safe

File: packages/editor-ui/src/components/banners/__tests__/__snapshots__/V1Banner.spec.ts.snap [hunk 4]
```diff
@@ -2,69 +2,209 @@
 
 exports[`V1 Banner > should render banner 1`] = `
 <div>
-  <n8n-callout
-    class="callout v1container"
+  <div
```
Comment (tomi) [inline packages/editor-ui/src/components/banners/__tests__/__snapshots__/V1Banner.spec.ts.snap:5]: This snapshot has changed quite drastically. Is that intentional?

File: pnpm-lock.yaml [hunk 5]
```diff
@@ -577,7 +577,7 @@ importers:
         version: 8.1.4(@types/react@18.0.27)(encoding@0.1.13)(prettier@3.2.5)(react-dom@18.2.0(react@18.2.0))(react@18.2.0)
       '@storybook/addon-interactions':
         specifier: ^8.1.4
-        version: 8.1.4(@jest/globals@29.6.2)(@types/jest@29.5.3)(jest@29.6.2(@types/node@18.16.16)(ts-node@10.9.2(@types/node@18.16.16)(typescript@5.6.2)))(vitest@1.6.0(@types/node@18.16.16)(sass@1.64.1)(terser@5.16.1))
+        version: 8.1.4(@jest/globals@29.6.2)(@types/jest@29.5.3)(jest@29.6.2(@types/node@18.16.16)(ts-node@10.9.2(@types/node@18.16.16)(typescript@5.6.2)))(vitest@1.6.0(@types/node@18.16.16)(jsdom@23.0.1)(sass@1.64.1)(terser@5.16.1))
```
Comment (tomi) [inline pnpm-lock.yaml:580]: Are these changes intentional? I don't see `jsdom` being used anywhere?

File: packages/editor-ui/src/components/banners/__tests__/__snapshots__/V1Banner.spec.ts.snap (hunk 4 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/banners/__tests__/__snapshots__/V1Banner.spec.ts.snap:5]: It's the side effect of using our render helper which has all the components and directives installed (this is what we want to introduce also in design system)

File: pnpm-lock.yaml (hunk 5 above)
Comment (cstuncsik) [reply pnpm-lock.yaml:580]: Just removed `@vue/test-utils` from design system package.json because `@testing-library/vue` has it's own and it's better to depend on that

File: packages/design-system/src/directives/n8n-html.test.ts (hunk 3 above)
Comment (cstuncsik) [reply packages/design-system/src/directives/n8n-html.test.ts:33]: Indeed (safe malicious :) ), I have changed the wording

**Review by tomi — DISMISSED** [review]
Thank you for addressing this 💘

**Review by tomi — DISMISSED** [review]

**Review by tomi — APPROVED** [review]
